roles/pulp_common/templates/settings.py.j2

```diff
@@ -1,3 +1,6 @@
 {% for setting, value in __pulp_common_merged_pulp_settings.items() %}
 {{ setting | upper }} = {{ value | string | capitalize if value | string in ["True", "False"] else value | to_json }}
 {% endfor %}
+{% if ansible_fips -%}
+ALLOWED_CONTENT_CHECKSUMS = {"sha1", "sha224", "sha256", "sha384", "sha512"}
```

Comment:
Ideally we would have this conditionally baked into `__pulp_common_pulp_settings_default` so users can still overrule it.
I can look into this if there is a demand for it.

Reply:
It will involve the use of `ternary` and not look nice.

Reply:
Isn't this a list? So shouldn't [the json](https://www.tutorialspoint.com/json/json_syntax.htm) have square brackets instead of curly braces?

Reply:
It's technically a set, but dynaconf can't specify a `set`, so maybe it should be a list here. What's important is that it's iterable.

That being said, I don't think the installer should blindly set this here. Users need to determine for themselves if they want sha1 or not. Can this be something users set normally just like any other pulp var expressed to the installer?

Reply:
The braces are really conflicting jinja2 syntax. So let's make it a list, an add it to the defaults, so the user can easily provide a personalized version.